### ISSUE TITLE
Random bugs with the homepage on mobile

### ISSUE THREAD

**Issue by noahsark769** [calblueprint/revolv#211]
1. Margin at the top is too large
2. Times new roman displays in the svg

**Comment by andylouisqin** [comment]
@noahsark769 this too should be fixed by #218 heh

**Comment by noahsark769** [comment]
```
            _
           /(|
          (  :
         __\  \  _____
       (____)  `|
      (____)|   |
       (____).__|
        (___)__.|_____
```
